src/main/java/org/spongepowered/mod/mixin/command/server/MixinCommandBlockLogic.java

```diff
@@ -0,0 +1,41 @@
+package org.spongepowered.mod.mixin.command.server;
+import java.util.ArrayList;
+import java.util.List;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.IChatComponent;
+import org.spongepowered.api.text.message.Message;
+import org.spongepowered.api.util.annotation.NonnullByDefault;
+import org.spongepowered.api.util.command.CommandSource;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+@NonnullByDefault
+@Mixin(value=net.minecraft.command.server.CommandBlockLogic.class,remap=false)
+public abstract class MixinCommandBlockLogic implements ICommandSender, CommandSource {
+	@Shadow public abstract void addChatMessage(IChatComponent icc);
+	@Override
+	public void sendMessage(String... messages) {
+		String s=new String();
+		for(String text:messages){
+			s += text;
```

Comment:
You need to do that for the other 2 methods as well.
Also, you must format the code according to the [Sponge contributing guidelines](https://docs.spongepowered.org/en/latest/devs/guidelines/).